src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -209,6 +254,8 @@ def __init__(
                 "Set `max_generated_tokens` to 1 to support that scenario."
             )
 
+        self._use_session_ids = False
```

Comment:
can you explain more what this is used/defaulted for?

Reply:
To handle the session_ids management in the pipeline context. But given that we want to completely outsource session_id management to the `NLDecoderEngine`, this is probably not needed!